AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginIntegrationTests/TestSupport/SyncEngineIntegrationV2TestBase.swift

```diff
@@ -0,0 +1,127 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import XCTest
+
+import AmplifyPlugins
+import AWSMobileClient
+
+@testable import Amplify
+@testable import AmplifyTestCommon
+@testable import AWSDataStoreCategoryPlugin
+
+class SyncEngineIntegrationV2TestBase: DataStoreTestBase {
+
+    static let amplifyConfigurationFile = "testconfiguration/AWSDataStoreCategoryPluginIntegrationV2Tests-amplifyconfiguration"
+
+    static let networkTimeout = TimeInterval(180)
+    let networkTimeout = SyncEngineIntegrationTestBase.networkTimeout
+
+    // Convenience property to obtain a handle to the underlying storage adapter implementation, for use in asserting
+    // database behaviors. Full of force-unwrapped badness.
+    // swiftlint:disable force_try
+    // swiftlint:disable force_cast
+    var storageAdapter: SQLiteStorageEngineAdapter {
+        let plugin = try! Amplify.DataStore.getPlugin(for: "awsDataStorePlugin") as! AWSDataStorePlugin
+        let storageEngine = plugin.storageEngine as! StorageEngine
+        let storageAdapter = storageEngine.storageAdapter as! SQLiteStorageEngineAdapter
+        return storageAdapter
+    }
+    // swiftlint:enable force_try
+    // swiftlint:enable force_cast
+
+    override func setUp() {
+        super.setUp()
+
+        continueAfterFailure = false
```

Comment:
nit: why do we need this?

Reply:
This was copied from the `SyncEngineIntegrationTestBase` and turns out the variable is not being used, removing it from both places

Reply:
this was copied from `SyncEngineIntegrationTestBase`, the doc about this property:
```
* @property continueAfterFailure
     * Determines whether the test method continues execution after an XCTAssert fails.
     *
     * By default, this property is YES, meaning the test method will complete regardless of how many
     * XCTAssert failures occur. Setting this to NO causes the test method to end execution immediately
     * after the first failure occurs, but does not affect remaining test methods in the suite.
     *
     * If XCTAssert failures in the test method indicate problems with state or determinism, additional
     * failures may be not be helpful information. Setting `continueAfterFailure` to NO can reduce the
     * noise in the test report for these kinds of tests.
```
so by setting it to NO, the test will end execution immediately instead of running beyond the first assert. This seems like a preference at the time the test base was first written.